server/src/test/java/io/spine/server/entity/EventPlayingEntityShould.java

```diff
@@ -17,225 +17,57 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+
 package io.spine.server.entity;
 
-import com.google.protobuf.Message;
 import com.google.protobuf.StringValue;
 import io.spine.core.Event;
 import io.spine.core.EventEnvelope;
-import io.spine.core.Version;
 import io.spine.server.command.TestEventFactory;
+import io.spine.server.event.EventStream;
 import io.spine.validate.StringValueVBuilder;
-import io.spine.validate.ValidatingBuilder;
 import org.junit.Test;
 
-import static com.google.common.collect.Lists.newArrayList;
 import static io.spine.test.TestValues.newUuidValue;
-import static io.spine.test.Tests.assertHasPrivateParameterlessCtor;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 /**
- * @author Alex Tymchenko
+ * @author Dmytro Dashenkov
  */
 public class EventPlayingEntityShould {
```

Comment:
I think that's a `TransactionalEventPlayerShould` now, as it's the one that fails to play events with no transaction and it's the one to delegate the execution to `tx` when playing.